ddl/db_change_test.go

```diff
@@ -833,6 +838,7 @@ func (s *testStateChangeSuiteBase) runTestInSchemaState(c *C, state model.Schema
 			return
 		}
 		for _, sqlWithErr := range sqlWithErrs {
+			forceReloadDomain(se)
```

Comment:
Could we put it after line839? I think it is enough for us to execute `forceReloadDomain` once in each schema state.